tmail-backend/blob/secondary-blob-store/src/test/java/com/linagora/tmail/blob/secondaryblobstore/SecondaryBlobStoreDAOTest.java

```diff
@@ -378,15 +378,6 @@ public void saveInputStreamShouldThrowOnIOException() {
             .isInstanceOf(IOException.class);
     }
 
-    @Test
-    @Override
-    public void saveShouldThrowWhenNullData() {
-        BlobStoreDAO store = testee();
-
-        assertThatThrownBy(() -> Mono.from(store.save(TEST_BUCKET_NAME, TEST_BLOB_ID, (byte[]) null)).block())
-            .isInstanceOf(ObjectStoreException.class);
```

Comment:
With new James code we can respect the original Contract (`ReadSaveBlobStoreDAOContract`)
Don't need override it 

// the original contract is check `.isInstanceOf(NullPointerException.class);`

Reply:
CC: @hungphan227